### DISCUSSION TITLE
Dependency conflict related to protobuf

### DISCUSSION THREAD

**Discussion by LSnyd** [NVIDIA/NVFlare#772] [Q&A] [answered]
Hi all, 
I've come across a dependency conflict as NVFlare requires protobuf==3.20, while the new tensorboard 2.10. requires protobuf<3.20. I ran into some errors using conda because of that. Might be helpful to add this to the requirements.txt :)

**Comment by YuanTingHsieh** [comment, marked as answer]
Hi @LSnyd thanks for this report!

Our current release of NVFlare has to use protobuf==3.20.1.
Can you have 2 virtual environment?
One with NVFlare and old version of tensorboard.
And the other use the new version of tensorboard.

We could think of maybe downgrade to lower version of protobuf if this new version has a lot of problems.

**Comment by LSnyd** [comment]
Hi @YuanTingHsieh, 
for me it was sufficient to downgrade tensorboard to version 2.9. This solved the problem.
